src/nvim/memory.c

```diff
@@ -149,7 +149,7 @@ void *xmallocz(size_t size)
 {
   size_t total_size = size + 1;
   if (total_size < size) {
-    OUT_STR(_("Vim: Data too large to fit into virtual memory space\n"));
+    mch_errmsg(_("Vim: Data too large to fit into virtual memory space\n"));
```

Comment:
Since most of these messages are still prefixed with "Vim:", I'd rather leave it like this for now and change in another PR